### PR TITLE
New Recipe: porta v1.4.1

### PR DESCRIPTION
**Pull request by bdoolittle**
Adding build_tarballs.jl for [porta](http://porta.zib.de/).  Script built by BinaryBuilder.run_wizard(). All platforms supported.

### CODE REVIEW

File: P/porta/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    cp julia-porta/gnu-make/bin/valid.exe ../destdir/bin/valid.exe;
+elif [[ $target == i686-w64-mingw32 ]]; then
+    make -C julia-porta/gnu-make/ CC=i686-w64-mingw32-gcc;
+    cp julia-porta/gnu-make/bin/xporta.exe ../destdir/bin/xporta.exe;
+    cp julia-porta/gnu-make/bin/valid.exe ../destdir/bin/valid.exe;
+else
+    make -C julia-porta/gnu-make/;
+    cp julia-porta/gnu-make/bin/xporta ../destdir/bin/xporta;
+    cp julia-porta/gnu-make/bin/valid ../destdir/bin/valid;
+fi
```
Comment (giordano) [inline P/porta/build_tarballs.jl:30]: We have some [environment variables](https://juliapackaging.github.io/BinaryBuilder.jl/dev/build_tips/#Automatic-environment-variables-1) predefined in the build environment that will make much simpler:
```suggestion
mkdir -p ${bindir}
make -C julia-porta/gnu-make/ CC=${CC}
cp julia-porta/gnu-make/bin/xporta${exext} ${bindir}
cp julia-porta/gnu-make/bin/valid${exext} ${bindir}
```

Comment (bdoolittle) [reply P/porta/build_tarballs.jl]: Hey, thanks for the tip! The script is much cleaner with the environment variables. I'll keep this in mind in the future.

I corrected the changes in a separate commit  as the suggestion dropped an 'e' in `${exeext}`.

File: P/porta/build_tarballs.jl [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "porta"
```
Comment (giordano) [inline P/porta/build_tarballs.jl:5]: Sorry, last request: as far as I can see the project is called [PORTA](http://porta.zib.de/), all uppercase:
```suggestion
name = "PORTA"
```
You should also rename the directory to exactly match the new name.  After this, we're good to go!

File: P/porta/build_tarballs.jl (hunk 1 above)
Comment (giordano) [reply P/porta/build_tarballs.jl]: Good catch!  I always miss the extra "e" :sweat_smile:

File: P/porta/build_tarballs.jl (hunk 2 above)
Comment (bdoolittle) [reply P/porta/build_tarballs.jl]: Done. I appreciate your help!

**Comment by giordano** [comment]
Thank you so much for your contribution!

File: P/porta/build_tarballs.jl (hunk 2 above)
Comment (giordano) [reply P/porta/build_tarballs.jl]: You're welcome :slightly_smiling_face:

**Pull request merged**